### ISSUE TITLE
Can't reproduce results

### ISSUE THREAD

**Issue by Fantaztig** [aadilh/blogs#2]
Hi! 

I just tried to reproduce your results on ubuntu 18.04 by pulling your code and executing it.
However, the results are not even close, discriminator loss seems so settle at about 1k iterations at 1.3 and gen loss at about 0.6 without any real changes for the next 8k iterations.
Could you give me more details on your setup like OS, python and tensorflow version etc?

Kind regards,
David

**Comment by aadilh** [comment]
Hi @Fantaztig when it comes to the GANs loss functions do not tell the full picture. Did you actually check the plot of the results being generated and verified that results are not ok? 
Please follow the detailed instructions provided [here](https://blog.paperspace.com/implementing-gans-in-tensorflow/)
